src/ansys/aedt/core/visualization/post/post_common_3d.py

```diff
@@ -1753,14 +1753,23 @@ def create_fieldplot_volume(
 
         Returns
         -------
-        :class:``ansys.aedt.core.modules.solutions.FieldPlot``
-            Plot object.
+        :class:``ansys.aedt.core.modules.solutions.FieldPlot`` or bool
+            Plot object or ``False`` if it fails.
 
         References
         ----------
 
         >>> oModule.CreateFieldPlot
         """
+        assignment = self.modeler.convert_to_selections(assignment, True)
+
+        for a in assignment:
+            if a not in list(self.modeler.objects_by_name.keys()):
+                self.logger.error("{} does not exist in current design".format(a))
+                return False
+            elif self.modeler.objects_by_name[a].is_conductor and not self.modeler.objects_by_name[a].solve_inside:
+                self.logger.warning("Solve inside is unchecked for {} object.".format(a))
+                return False
```

Comment:
```suggestion
        for element in assignment:
            if element not in list(self.modeler.objects_by_name.keys()):
                self.logger.error(f"{element} does not exist in current design")
                return False
            elif self.modeler.objects_by_name[a].is_conductor and not self.modeler.objects_by_name[a].solve_inside:
                self.logger.warning(f"Solve inside is unchecked for {element}.")
                return False
```
Can you explain to me why there is a need for the `elif` case ? IMO, it's a bit strange to see this checking in a method named `create_fieldplot_volume`. I might lack knowledge and if that's the case, please enlighten me.